packages/labs/gen-wrapper-angular/package.json

```diff
@@ -2,7 +2,7 @@
   "private": true,
```

Comment:
Similarly gen-wrapper-angular is still private. I guess the patch changes are expected since we've had them previously in the CHANGELOG.md